web/themes/custom/drupalam/js/blog-page.js

```diff
@@ -34,12 +34,4 @@
     })
   }
 
-  function MasonryGrid() {
```

Comment:
Why we remove  `MasonryGrid()` function?

Reply:
I can't remember. Restored masonry grid on Blog posts listing page.

Reply:
I don't see it restored :-(